packages/@o3r/eslint-plugin/src/rules/utils.ts

```diff
@@ -21,13 +21,30 @@ export const defaultSupportedInterfaceNames = ['Configuration', 'NestedConfigura
  * @param interfaceDeclNode
  * @param supportedInterfaceNames
  */
-export const isExtendingConfiguration = (interfaceDeclNode: TSESTree.Node | undefined, supportedInterfaceNames: string[] = []) => {
+export const isExtendingConfiguration = (interfaceDeclNode: TSESTree.Node | undefined, supportedInterfaceNames: string[] = []): interfaceDeclNode is TSESTree.Node => {
   const supportedInterfaceNamesSet = new Set(supportedInterfaceNames.length > 0 ? supportedInterfaceNames : defaultSupportedInterfaceNames);
   return interfaceDeclNode?.type === TSESTree.AST_NODE_TYPES.TSInterfaceDeclaration
-    && interfaceDeclNode.extends?.some((ext) =>
+    && !!interfaceDeclNode.extends?.some((ext) =>
       ext.type === TSESTree.AST_NODE_TYPES.TSInterfaceHeritage
       && ext.expression.type === TSESTree.AST_NODE_TYPES.Identifier
       && supportedInterfaceNamesSet.has(ext.expression.name)
     );
 };
 
+/**
+ * Returns the comment above the node
+ * @param node
+ * @param sourceCode
+ */
+export const getNodeComment = (node: TSESTree.Node, sourceCode: TSESLint.SourceCode): TSESTree.Comment | undefined => {
+  const [comment] = node.parent?.type === TSESTree.AST_NODE_TYPES.ExportNamedDeclaration
+    ? sourceCode.getCommentsBefore(node.parent)
+    : sourceCode.getCommentsBefore(node);
+  return comment;
+};
+
+/**
+ * Wraps `commentValue` into a comment
+ * @param commentValue
+ */
+export const createCommentString = (commentValue: string) => `/*${commentValue}*/`;
```

Comment:
Maybe you should sanitize a bit the content:

```suggestion
export const createCommentString = (commentValue: string) => `/*${commentValue.replace(/\*\//g, '*\\/')}*/`;
```